Update documentation definitions for operators (#2016)
FerretDB/FerretDB: website/docs/reference/operators/array-operators.md

## $all

*Syntax*: `{ <field>: { $all: [ <element1>, <element2>, ... <elementN> ] } }`

> Use the `all` operator when you want to select documents that contain every single element in a specified array.

Reviewer: ```suggestion
Use the `$all` operator when you want to select documents that contain every single element in a specified array.
```

> The `$all` operator matches documents where a particular field is an array containing all the elements given in the query.

Reviewer: `$all` operator selects documents with an array field that contains all elements from the given query.

Some suggestion, not exact wording.

## $elemMatch

*Syntax*: `{ <field>: { $elemMatch: { <condition1>, <condition2>, ... <conditionN>} } }`

> To select documents in a specified array field that matches at least one of several listed query conditions, use the `$elemMatch` operator.

Reviewer: ```suggestion
To select documents in a specified array field where one or more elements match all listed query conditions, use the `$elemMatch` operator.
```

Trying to highlight that we want to find one element or more to match all conditions.
